contracts/smart-contract-wallet/SmartAccount.sol

```diff
@@ -176,7 +181,7 @@ contract SmartAccount is
     // init
     // Initialize / Setup
     // Used to setup
-    function init(address _owner, address _handler) public override initializer { 
+    function init(address _owner, address _handler) public override { 
         require(owner == address(0), "Already initialized");
         require(_owner != address(0),"Invalid owner");
         require(_handler != address(0), "Invalid Fallback Handler");
```

Comment:
we can move the _handler check to `setFallbackHandler`, as `setFallbackHandler` still doesn’t have any 0 address check

Reply:
let me check

Reply:
done as part of code security: check this commit https://github.com/bcnmy/scw-contracts/commit/285958386f1c7baac0a5f72b48d5f9b78f71f573

Reply:
cool

Reply:
i removed it in that commit..